Sources/HAP/Utils/TLV8.swift

```diff
@@ -1,15 +1,24 @@
 import Foundation
 
-typealias PairTagTLV8 = [PairTag: Data]
+typealias PairTagTLV8 = (PairTag, Data)
```

Comment:
This would keep it more in spirit of the original type and also results in less changes: `typealias PairTagTLV8 = [(PairTag, Data)]`

Reply:
I preferred to use the type PairTagTLV8 to refer to a single pair, and [PairTagTLV8] or PairTagTLV8Array to refer to a sequence of PairTag’s, however I will rename as you requested.